test/plugins/auto-load.spec.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiAsPromised);
+
+const expect = chai.expect;
+const _ = require('lodash');
+const simple = require('simple-mock');
+const StateMachineSkill = require('../../lib/StateMachineSkill');
+const autoLoad = require('../../lib/plugins/auto-load');
+const views = require('../views');
+const variables = require('../variables');
+const AutoLoadAdapter = require('../autoLoadAdapter');
+
+const adapter = new AutoLoadAdapter();
+
+describe('AutoLoad plugin', () => {
+  let event;
+
+  beforeEach(() => {
+    event = {
+      sesssion: {
+        user: {
+          userId: 'user-xyz',
+        },
+        new: true,
+      },
+      request: {
+        type: 'LaunchRequest',
+      },
+    };
+
+    simple.mock(AutoLoadAdapter.prototype, 'get')
```

Comment:
just what i was saying, we're "never" using the dynamodb getItem implementation, we're always mocking it. Let's remove the dynamodb dependency OR make this a separate package and actually provide the adapter for users so they can use it